DashAISoftware/DashAI
# Reorder files & delete unused components

# Summary


Improved the name of the page files, some files have been moved to folders to enhance organization, and unused components have been deleted.

## Type of change



- Refactoring.

## Changes



- Moved main components from `/tabs` folder into  `/pages` folder and improved their names.
- Moved all dataset related components into the `/datasets` folder.
- Removed unused components.
- In `ResponsiveAppBar.jsx` change the name of the page "data" into "datasets".

## How to Test


1. Go to `DashAI/DashAI/front` and run `yarn build`
2. Go back to `DashAI/` and run `python -c "import DashAI;DashAI.run()"`
3. The app should work normally.

## Screenshots



"Data" page in the AppBar has been renamed to "Datasets".

![datasets](https://github.com/DashAISoftware/DashAI/assets/53657198/8ca89c25-03c7-442a-a7d9-7d6257adfd9a)
